OpenQA/Isotovideo/Utils.pm

```diff
@@ -20,7 +20,7 @@ use Mojo::URL;
 use Exporter 'import';
 use File::Spec;
 use Cwd;
-use bmwqemu 'diag';
+use bmwqemu;
```

Comment:
maybe it would be better to explicitly import nothing:
```
use bmwqemu ();
```
since the module exports two subs by default, which we don't use.

Reply:
Thank you. This is also making it more obvious what we want to ensure here. I was thinking about a comment in front but `use bmwqemu ();` is better :)